test-server.js

```diff
@@ -1,24 +1,27 @@
 const request = require('supertest');
 const expect = require('chai').expect;
 
-
-describe('test server', function () {
+describe('test server', () => {
     let server;
 
-    beforeEach(function () {
+    beforeEach(() => {
         server = require('./server.js');
     });
 
-    afterEach(function () {
+    afterEach(() => {
         server.close();
     });
 
     it('responds to /api/home/', (done) => {
-        request(server).get('/api/home/').expect(200, done);
+        request(server)
+            .get('/api/home/')
+            .expect(200, done);
     });
 
     it('404 other urls', (done) => {
```

Comment:
```suggestion
    it('should respond with 404 on other urls', (done) => {
```

Nit: prefer the convention of starting tests with should